addons/hr_expense/models/product_product.py

```diff
@@ -16,7 +16,7 @@ def _compute_standard_price_update_warning(self):
         mapp = {row['product_id'][0]: row['unit_amount'] for row in undone_expenses}
         for product in self:
             product.standard_price_update_warning = False
-            if undone_expenses:
+            if product in mapp.keys():
```

Comment:
```suggestion
            if product._origin.id in mapp.keys():
```
The reason runbot is red

Reply:
Damn thanks for the super fast review :smile: